module/os/linux/zfs/zpl_xattr.c

```diff
@@ -1002,6 +999,20 @@ zpl_set_acl(struct inode *ip, struct posix_acl *acl, int type)
 	return (error);
 }
 
+#ifndef HAVE_SET_ACL
+static
+#endif
```

Comment:
It looks like what we want to do is only define `zpl_set_acl()` when `HAVE_SET_ACL` is defined.  It also shouldn't need to be static since internally you're calling the `_impl()` version which is nice.  That should sort out the CentOS 7 build failure.

```
module/zfs/../os/linux/zfs/zpl_xattr.c:1010:1: error: 'zpl_set_acl' defined but not used [-Werror=unused-function]
 zpl_set_acl(struct inode *ip, struct posix_acl *acl, int type)
 ^
cc1: all warnings being treated as errors
```

Reply:
Whoops, totally missed that failure.

Corrected.